ipcimpl/sm.cpp

```diff
@@ -6,13 +6,21 @@ partial SmService {
 }
 */
 
+static bool smInitialized = false;
+
 uint32_t SmService::Initialize() {
+	smInitialized = true;
 	return 0;
 }
 
 #define SERVICE(str, iface) do { if(name == (str)) { svc = buildInterface(iface); return 0; } } while(0)
 
 uint32_t SmService::GetService(IN ServiceName _name, OUT shared_ptr<IPipe>& svc) {
+	if(!smInitialized) {
```

Comment:
Libtransistor assumes `sm:h`. I think that if we're going to "patch" `sm:h` in Mephisto, it should be behind a config option.